src/shared/components/palette/history/event-log/event-log-table.component.html

```diff
@@ -23,7 +23,7 @@
       </ng-container>
       <span *ngIf="event.state_id === 'Draft'">{{event.event_name}}</span>
       <div class="tooltip" *ngIf="significantItemExist(event)">
-        <a href="{{getSignificantItemUrl(event)}}" target="_blank">
+        <a href="{{getSignificantItemUrl(event)}}" target="_blank" [attr.aria-label]="getSignificantItemDesc(event)">
```

Comment:
We have two different binding patterns in here: `href="{{ ... }}"` and `[attr.aria-label]="..."`.

The former would be ideal in both instances as it only happens once and isn't a function binding.